container.go

```diff
@@ -62,6 +62,7 @@ type Container interface {
 	CopyFileToContainer(ctx context.Context, hostFilePath string, containerFilePath string, fileMode int64) error
 	CopyFileFromContainer(ctx context.Context, filePath string) (io.ReadCloser, error)
 	GetLogProductionErrorChannel() <-chan error
+	Hostname(ctx context.Context) (string, error)
```

Comment:
Introducing a new method in the interface could be seen as a breaking change, so not sure if we should do this here.

Of course, if it's needed it's needed

Reply:
I think we should just return the whole inspect struct 🤔

Reply:
Then we should rename the method to `Config`